.github/pull_request_template.md

```diff
@@ -2,7 +2,7 @@
 <!-- Please delete options that are not relevant. -->
 
 - Fixes #<!--issue-no-->
-- Issue/Addition to code, goes here.
+- Addition to code.
```

Comment:
I think Issue should be inculded in this line too, just incase the issue isn't really made as a issue in the repo if you get what I mean?

Reply:
I think it's fine. Someone can add the issue to `Fixes #` if they really want to.

Reply:
Yeah, fair enough.